salt/pillar/elife-xpub.sls

```diff
@@ -16,6 +16,7 @@ elife_xpub:
         base_url: fake_pubsweet_baseurl
     s3:
         bucket: fake_bucket
+    deployment_target: production
```

Comment:
this is a strange default however.

Reply:
The value here should only be used in Vagrant VMs, so it should be more similar to development than production.

Taking a look at https://github.com/elifesciences/elife-xpub/tree/develop/config, `development` or `test`?

Reply:
I suggest using test, until we've created a CI config file, and then we can change this `ci`